### DISCUSSION TITLE
Allow setting from, text part and headers in Flow Send email Operation

### DISCUSSION THREAD

**Discussion by filiphanes** [directus/directus#17114] [Feature Requests]
Hi,
I would like to
- set custom from name and address
- custom email headers
- add text/plain part for better compatibility
All with templating support, of course.

I know it can be done in Run script Operation, but this would be nicer, when we already have Send email operation.

And thanks for this great project.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
